tests/containers/bci_test.pm

```diff
@@ -65,7 +65,7 @@ sub reset_engines {
     my ($self, $current_engine) = @_;
```

Comment:
I think this makes perfect sense to be moved into a `bci_prepare` rather than keeping it in `bci_test`

Reply:
We can, let's do this in a later step, we need to release the BCI containers in the pipeline first.

Reply:
After looking at it more closely, I think we should leave it in the `bci_test.pm` module, as it stops docker when podman is being tested. We cannot do this globally, as we are testing docker and podman within one test run.

Reply:
It's unfortunately needed in both test runs, but we can move it to a library function.

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17527